src/app/website/company-home/feature-summary/feature-summary.component.html

```diff
@@ -0,0 +1,29 @@
+<section class="featuresummary">
+  <div class="row featuresummary__content"
+       *ngFor="let data of displayModel?.featuresSummarySections">
```

Comment:
when you are using smart and dumb components then the questionmark should not be used. you should pass data from smart component only when it it available